integration_tests/pages/courtCaseTaskListPage.ts

```diff
@@ -7,8 +7,6 @@ export default class CourtCaseTaskListPage extends Page {
 
   appearanceInformationLink = (): PageElement => cy.get('a:contains("Add appearance information")')
 
-  sentencesLink = (): PageElement => cy.get('a:contains("Add offences")')
```

Comment:
This sentencesLink referenced the same element as the offences link, so just removed it and refactored  the calling test to use offencesLink